.env.example

```diff
@@ -20,5 +20,5 @@ POSTGRES_PASSWORD=helpwave
 POSTGRES_DB=helpwave
 
 # For swagger definition
-BASE_URL=localhost:3000
-UNSECURE=enabled
+BASE_URI=localhost:3000
```

Comment:
I'd argue our base url is `https://api.helpwave.de/v1`, at least that what I associate with a URL. Here we set the BASE_URI variable to only the host part.

Reply:
Yeah. The url will contain the protocol. We dont need the protocol here.
But host:port is a valid uri. Thats why I changed the name,

Reply:
I don't know why you want to name the environment variable, which we use to set a "Host" property of swaggerinfo, BASE_URI, even though that is a lot more ambiguous. I get that it is useful to have a BASE_URI variable, but this is not it.